generators/client-2/templates/src/main/webapp/app/admin/health/health.html

```diff
@@ -12,19 +12,18 @@ <h2 translate="health.title">Health Checks</h2>
                 <th class="col-md-7" translate="health.table.service">Service Name</th>
                 <th class="col-md-2 text-center" translate="health.table.status">Status</th>
                 <th class="col-md-2 text-center" translate="health.details.details">Details</th>
-
             </tr>
         </thead>
         <tbody>
-            <tr ng-repeat="health in vm.healthData">
-                <td><% if (enableTranslation){ %>{{'health.indicator.' + vm.baseName(health.name) | translate}}<% } else { %><span class="text-capitalize">{{ vm.baseName(health.name) }}</span><% } %> {{vm.subSystemName(health.name)}}</td>
```

Comment:
can we add a dummy pipe now and put a todo there? So that in future if something comes to fill it here will be a minimal change ?